src/har/request.cr

```diff
@@ -141,6 +143,8 @@ module HAR
       )
       cookies.each do |cookie|
         request.cookies << cookie.to_http_cookie
+      rescue ex
+        Log.error(exception: ex) { "Invalid cookie: #{cookie.inspect}" }
```

Comment:
I'd use `warn` level, since the cookie is silently ignored.

```suggestion
        Log.warn(exception: ex) { "Invalid cookie: #{cookie.inspect}" }
```